Adding draft of processes for CGIAR module 1

Mttbnchtt opened the pull request:
As explained in #169, I add the processes for the CGIAR module 1.

Mttbnchtt committed:
Addition of process to reduce poverty (right branch)

Mttbnchtt committed:
Added draft of processes for CGIAR M1

Mttbnchtt committed:
I rephrased a few labels as imperatives

Mttbnchtt committed:
Hierarchy of classes for processes concerning CGIAR module 1

Mttbnchtt commented:
@marieALaporte @pbuttigieg 
I created a hierarchy of process classes. Let me know what you think.

Mttbnchtt committed:
More elaborate hierarchy of processes

Mttbnchtt committed:
Stylistic improvement

marieALaporte commented on src/sdgio-edit.owl:
```diff
@@ -1608,6 +1638,235 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
+# Class: <http://purl.unep.org/sdg/SDGIO_00020335> (planned process to manage resources)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.unep.org/sdg/SDGIO_00020335> "A planned process during which an agent causes or contributes to the utlization or protection of resources.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000116> <http://purl.unep.org/sdg/SDGIO_00020335> "The SDGIO contains an entry \"resource\". It is appropriate to link that entry to this entry. However, the entry \"resource\" does not have a PURL.")
```
@Mttbnchtt not sure what you meant here. http://purl.unep.org/sdg/SDGIO_00000013 is the PURL for 'resource' in SDGIO

Mttbnchtt replied:
Thanks for that. I changed the editor note for the process. I did not see, and still do not see, that PURL in Protege. I am probably looking in the wrong place. Where did you find the PURL?

marieALaporte replied:
You click on a class and press command and U (on mac), then click on the box "see full URI" something like that

Mttbnchtt replied:
I linked "planned process to manage resouces" to "resource" via the axiom "has participant some resource".

marieALaporte commented:
A link between the class 'planned process to manage resources' and 'resource' could be added, using 'has participant'. The resource can be refined in the different subclasses.

Regarding the 'process of being educated' and 'process of educating', they could be linked somehow to the 'education process'. Also,  'process of being educated' and 'process of educating' are kind of inverse processes.

Mttbnchtt commented on src/sdgio-edit.owl:
```diff
@@ -1608,6 +1638,235 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020321> "process to enhance the institutional capacity of research organizations that partner with CGIAR"@en)
+SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020321> <http://purl.unep.org/sdg/SDGIO_00020316>)
+
+# Class: <http://purl.unep.org/sdg/SDGIO_00020322> (process to enhance - through training or exchange - the individual capacity in research organizations that partner with CGIAR)
```
Yes, thanks very much for that. I fixed them.

Mttbnchtt committed:
Fixed mistakes found by Marie-Angélique

Mttbnchtt commented:
Thanks @marieALaporte for the comments. I fixed the mistakes that you found.

Mttbnchtt committed:
Adding "has participant" resource to planned process to manage resources

Mttbnchtt committed:
Added natural resource, axioms, and fixed position of planned process to improve human diet

Mttbnchtt committed:
Added diet, human diet, and axioms to related them process to improve diet and human diet

Mttbnchtt committed:
Added notes to Diet and Human Diet

Mttbnchtt committed:
Remove problematic axiom "is about some diet"

Mttbnchtt committed:
removing other problematic axiom for human diet

Mttbnchtt committed:
Axioms for diet related processes

Mttbnchtt committed:
Added axioms for processes concerning climate change

Mttbnchtt committed:
Added 'has component process' axioms

Mttbnchtt committed:
Axioms to related process to respond to climate change to pollution control and monitorning

marieALaporte commented:
@pbuttigieg @Mttbnchtt the 'inclusive wealth' class is also inconsistent, although I am not sure when the 'has quality' axiom has been introduced in the ontology and why the travis check is passing.

'has quality' cannot be used with dispositions, only with qualities

<img width="1193" alt="Capture d’écran 2019-06-07 à 09 49 12" src="https://user-images.githubusercontent.com/5664149/59089171-bf609680-8909-11e9-91a5-5859d8b2b4c9.png">

Mttbnchtt commented:
Thanks @marieALaporte  for spotting that inconsistency. It is very important. That also reminds me that I have yet to clarify what are the new items that I introduced. Those are all and only the following:
- those whose labels begin with "planned process to"
- diet
- human diet.

marieALaporte commented:
@Mttbnchtt I am going to create an issue for it as it should not be solved in this PR you're right

Mttbnchtt committed:
Added axiom to response to climate change

Mttbnchtt committed:
Change label for SLO: from outcome to objective

Given how this entity is actually used, it better fits the SDGIO to call it “system level objective”.

Mttbnchtt committed:
Deleted some axioms

The superclass axioms hold for subclasses too.

Mttbnchtt committed:
Adjstuments to Diet and Human Diet

Mttbnchtt commented:
In issue #179 I explain that we are in the process of adding "organization" from ENVO to SDGIO. @pbuttigieg will finalize the import before merging this PR.

Mttbnchtt committed:
Added new terms to import from ENVO to SDGIO

pbuttigieg committed:
Merge branch 'master' into issue-169

Mttbnchtt committed:
Updated annotations of "process to manage resources"

Mttbnchtt committed:
has input and output for process to improve diet

Mttbnchtt committed:
Parthood axioms for CGIAR objectives

Mttbnchtt committed:
Causal influences from planned process to manage resources

Mttbnchtt committed:
Causal influences from other planned processes

Mttbnchtt committed:
Remove wrongly used "causally influences" axiom

Mttbnchtt committed:
Rephrased a CGIAR objective

Mttbnchtt committed:
Moved "organization" under "system"

pbuttigieg commented on src/imports/envo_terms.txt:
```diff
@@ -94,4 +94,11 @@ http://purl.obolibrary.org/obo/ENVO_00000006 # second-order administrative regio
+http://purl.obolibrary.org/obo/ENVO_01000906 # aquaculture process
+http://purl.obolibrary.org/obo/ENVO_01001442 # agriculture
+http://purl.obolibrary.org/obo/ENVO_01001451 # usage of an environment for business
+http://purl.obolibrary.org/obo/FOODON_03470107 # food preservation process
```
Careful - only use the terms.txt file to import terms from the named ontology. This line will import ENVO's import of FOODON's term, which creates a potentially (inevitably) messy dependency chain.

Mttbnchtt replied:
Yes, I understand now. Should I remove the foodon term?

pbuttigieg committed:
added foodon import, removed import chain in envo_terms.txt

pbuttigieg committed:
added foodon import to catalog and in editors file

pbuttigieg committed:
refreshing obi import

pbuttigieg committed:
imported human population and activity terms from PCO

pbuttigieg committed:
added some nutrition-related classes from FOODON

pbuttigieg committed:
trying to fix foodon orphans

pbuttigieg commented on src/sdgio-edit.owl:
```diff
@@ -1608,6 +1645,291 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020315> "planned process to apply or improve institutional policies to increase the capacity of CGIAR partners to take actions concerning agriculture"@en)
+SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020315> <http://purl.unep.org/sdg/SDGIO_00020330>)
+
+# Class: <http://purl.unep.org/sdg/SDGIO_00020316> (planned process to increase the research capacity of CGIAR national partners and beneficiaries)
```
Same as the case above, generalise this beyond the CG. You can add the CG case as an instance.

Mttbnchtt replied:
Done, hopefully.

pbuttigieg commented on src/sdgio-edit.owl:
```diff
@@ -1608,6 +1645,291 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020313> "planned process to adapt to climate change"@en)
+SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020313> <http://purl.unep.org/sdg/SDGIO_00020325>)
+
+# Class: <http://purl.unep.org/sdg/SDGIO_00020315> (planned process to apply or improve institutional policies to increase the capacity of CGIAR partners to take actions concerning agriculture)
```
You can add "planned process to apply or improve institutional policies to increase the capacity of CGIAR partners to impact agricultural systems"  as an instance of the revised class.

Mttbnchtt replied:
I did this. I do not know how to make the label shorter though.

pbuttigieg commented on src/sdgio-edit.owl:
```diff
@@ -1608,6 +1645,291 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
+SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020316> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002018> <http://purl.unep.org/sdg/SDGIO_00020323>))
+SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020316> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002018> <http://purl.unep.org/sdg/SDGIO_00020324>))
+
+# Class: <http://purl.unep.org/sdg/SDGIO_00020317> (planned process to improve the forecasting of impacts of climate change)
```
Revised label. 

More importantly, this has a circular definition - we can either 1) add climate change forecasting  as a class, cross-link it or 2) define it in this def or 3) both.

I vote for 3).

Mttbnchtt replied:
For now I did 2). I agree that 3) would be better. I suppose that forecasting should be a planned process. I will work on that.

pbuttigieg commented on src/sdgio-edit.owl:
```diff
@@ -1608,6 +1645,291 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020317> "planned process to improve the forecasting of impacts of climate change"@en)
+SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020317> <http://purl.unep.org/sdg/SDGIO_00020325>)
+
+# Class: <http://purl.unep.org/sdg/SDGIO_00020318> (planned process to improve the capacity of women to participate in decision-making)
```
no hyphen in decision making. 

Please create an issue for us to import the needed classes to compose "human female". 
Also, please add a PCO issue to create a class for "population of human females".
The definition is circular, please revise.

Mttbnchtt replied:
Done, hopefully.

pbuttigieg commented on src/sdgio-edit.owl:
```diff
@@ -1608,6 +1645,291 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020318> "planned process to improve the capacity of women to participate in decision-making"@en)
+SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020318> <http://purl.unep.org/sdg/SDGIO_00020335>)
+
+# Class: <http://purl.unep.org/sdg/SDGIO_00020319> (planned process to improve the capacity of young people to participate in decision-making)
```
As above, we'll need to import semantics to compose "young human" and PCO needs a "population of young humans" class. PATO has the quality [young](http://purl.obolibrary.org/obo/PATO_0000309), which both should use.

pbuttigieg replied:
definition is circular, please revise.

Mttbnchtt replied:
done, hopefully

pbuttigieg reviewed (changes requested):
Hi @Mttbnchtt I've done a partial review as there are recurring issues which you can probably address globally before the next review. These include removing CG specific classes (those should be instances if we want the objective logic to work well) and rephrasing defs to remove circularity (as we still need to add many of the components mentioned in the defs, but that's beyond your timeframe).

Please revise all the additions with the above in mind and initiate another review.

#### Questions
* Why were direct changes made to the obi_import.owl file? This file should only be updated using the makefile routines. Best to roll back these changes unless they were intentional (if so please explain)
 * @Mttbnchtt notes that he didn't intend to change this file. I'm refreshing the OBI import to patch this.

#### Changes

* set "diets" to participants rather than both inputs and outputs
* rephrased definitions and axioms to reference human populations where necessary
* rephrasing for grammar/clarity

#### Related issues
* https://github.com/ontodev/robot/issues/515
* https://github.com/information-artifact-ontology/IAO/issues/214

pbuttigieg committed:
changes in aid of review for https://github.com/SDG-InterfaceOntology/sdgio/pull/170#pullrequestreview-251841342

Mttbnchtt committed:
Updated 'planned process to respond to global challenges'

Mttbnchtt commented on src/sdgio-edit.owl:
```diff
@@ -1608,6 +1645,291 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020337> "planned process to achieve or maintain fairness"@en)
+SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020337> <http://purl.obolibrary.org/obo/OBI_0000011>)
+
+# Class: <http://purl.unep.org/sdg/SDGIO_00020338> (planned process to respond to challenges)
```
I modified that entry, which is now called 'planned process to respond to global challenges'.

Mttbnchtt commented on src/sdgio-edit.owl:
```diff
@@ -1608,6 +1645,291 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020308> "planned process to reduce poverty"@en)
+SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020308> <http://purl.unep.org/sdg/SDGIO_00020338>)
+
+# Class: <http://purl.unep.org/sdg/SDGIO_00020309> (planned process to improve food security)
```
Great, thanks.

Mttbnchtt commented on src/sdgio-edit.owl:
```diff
@@ -1608,6 +1645,291 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020309> "planned process to improve food security"@en)
+SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020309> <http://purl.unep.org/sdg/SDGIO_00020326>)
+
+# Class: <http://purl.unep.org/sdg/SDGIO_00020310> (planned process to improve nutrition security)
```
Yes, some FOODON classes may be useful. We should have a better idea of what this process is for CGIAR to pick up the right FOODON classes. I will work on this.

Mttbnchtt replied:
See issue #198.

Mttbnchtt commented on src/sdgio-edit.owl:
```diff
@@ -1608,6 +1645,291 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
+# Class: <http://purl.unep.org/sdg/SDGIO_00020311> (planned process to improve sustained access to natural resources through interventions in the agriculture systems)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://hdl.handle.net/10947/3865>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.unep.org/sdg/SDGIO_00020311> "A planned process during which an agent causes or contributes to either 1) the building of a more sustainable and efficient agriculture sector, 2) the restoration of degraded land and wa ...
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020311> <http://orcid.org/0000-0062-3253-4387>)
```
Well done. Thanks.

Mttbnchtt commented on src/sdgio-edit.owl:
```diff
@@ -1608,6 +1645,291 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
 AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020300> "CGIAR cross-cutting objective"@en)
 SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020300> <http://purl.unep.org/sdg/SDGIO_00000251>)
 
+# Class: <http://purl.unep.org/sdg/SDGIO_00020308> (planned process to reduce poverty)
```
This is a very hard definition. I had discussions about that but got nothing really useful. My main worry is that now the definition seems circular (roughly: reduction of poverty is ... rising population above the poverty line).

Mttbnchtt replied:
Just to clarify, poverty, in some UN documents, is considered not just dependent on material resources (especially money) available now. It also depends on vulnerability (how easy is it for one to lose essential resource), education, etc. Your idea of using "basic goods" may allow us to find a solution though. What about:

"A planned process during which an agent - through either economic, technological, political, or societal means - secures stable access to basic goods and services, employment, or other resources or opportunities by which a human population can satisfy the bare necessities of life."

pbuttigieg replied:
> the definition seems circular (roughly: reduction of poverty is ... rising population above the poverty line)
 
 There is an operational definition of the poverty line that is used by UN Stats, but I see your point. 

>"A planned process during which an agent - through either economic, technological, political, or societal means - secures stable access to basic goods and services, employment, or other resources or opportunities by which a human population can satisfy the bare necessities of life."

Satisfying the bare necessities of life is not always enough to escape an impoverished state. 


https://en.wikipedia.org/wiki/Poverty#Definitions_and_etymology
> United Nations: Fundamentally, poverty is the inability of having choices and opportunities, a violation of human dignity. It means lack of basic capacity to participate effectively in society. It means not having enough to feed and clothe a family, not having a school or clinic to go to, not having the land on which to grow one's food or a job to earn one's living, not having access to credit. It means insecurity, powerlessness and exclusion of individuals, households and communities. It means susceptibility to violence, and it often implies living in marginal or fragile environments, without access to clean water or sanitation.[26]
>
>World Bank: Poverty is pronounced deprivation in well-being, and comprises many dimensions. It includes low incomes and the inability to acquire the basic goods and services necessary for survival with dignity. Poverty also encompasses low levels of health and education, poor access to clean water and sanitation, inadequate physical security, lack of voice, and insufficient capacity and opportunity to better one's life.[27]

Could you follow up that Wiki article and find the latest UN / World Bank definitions?

The definition would then go in the direction of  "...by which a human population can maintain and improve their health, well-being, security, dignity, education, and ability to participate in society."

Mttbnchtt replied:
New iteration: A planned process during which an agent - through either economic, technological, political, or societal measures - causes or contributes to people acquiring and maintaining the capacity to appropriately take care of their health, well-being, security, dignity, education, and ability to participate in society.

Note: I say "acquiring and maintaining the capacity to appropriately take care etc." because there is no guarantee even for a rich that she will acquire or maintain her health, well-being, security, etc. 

Is that ok?

Mttbnchtt committed:
Updated commet to process to reduce poverty

Mttbnchtt committed:
Removed CGIAR classes and added CGIAR instances

Mttbnchtt committed:
Update process structure

Mttbnchtt committed:
Update process sructure /2

Mttbnchtt committed:
Update process structure /3

Mttbnchtt committed:
Update process structure /3

Mttbnchtt committed:
Update process structure /4

Mttbnchtt committed:
Update process structure /5

Mttbnchtt committed:
Update process structure /6

Mttbnchtt committed:
Update process structure /7

Mttbnchtt commented on src/sdgio-edit.owl:
```diff
@@ -1608,6 +1645,291 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020311> "planned process to improve sustained access to natural resources through interventions in the agriculture systems"@en)
+SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020311> <http://purl.unep.org/sdg/SDGIO_00020327>)
+
+# Class: <http://purl.unep.org/sdg/SDGIO_00020312> (planned process to mitigate climate change)
```
Done.

Mttbnchtt committed:
Update process structure /8

Mttbnchtt committed:
Corrections

Mttbnchtt committed:
Improve labels

Mttbnchtt committed:
Improved hierarchy

Mttbnchtt committed:
Updated one label and one definition

pbuttigieg commented on src/sdgio-edit.owl:
```diff
@@ -158,6 +159,62 @@ Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020232>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020328>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020329>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020331>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020334>))
```
@Mttbnchtt make an issue to import http://purl.obolibrary.org/obo/ENVO_01001170 into the next revision and add this class and related classes underneath it. The definitions would need to be revised accordingly. Definition revised.

Mttbnchtt replied:
Done, issue #194.

pbuttigieg commented on src/sdgio-edit.owl:
```diff
@@ -158,6 +159,62 @@ Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020232>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020329>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020331>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020334>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020335>))
```
Definition revision. Protection is a specification of management, not all management will have that as a goal.

Mttbnchtt replied:
Right.

pbuttigieg commented on src/sdgio-edit.owl:
```diff
@@ -158,6 +159,62 @@ Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020232>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020331>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020334>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020335>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020337>))
```
@Mttbnchtt This class makes no sense to me. I think the language here is unintelligible and mostly "fluff". Is there anything more concrete you can find?

Mttbnchtt replied:
I will delete it. I inserted a while ago to deal with CGIAR process to improve gender equity and youth inclusion. On Jun 4 2019 on Skype, discussing an earlier version of it, you suggested to keep it (but without knowing the definition). The definition seems roughly correct to me. The language of all-things-considered right tries to express the idea that fairness does not mean to respect every right (as rights may conflict), but to find an appropriate balance concerning which rights to respect and in which measure. However, the class is not doing any work now and it is probably better to delete. If you want me to reinstate it, I can do that and we can figure out how to define it.

pbuttigieg commented on src/sdgio-edit.owl:
```diff
@@ -158,6 +159,62 @@ Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020232>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020334>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020335>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020337>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020338>))
```
Rephrased def. This is a slippery class too, but I think the definition is general enough as a starting point.

Does the "global community" refer to the global population of humans? the community of nation states? organisations?

Mttbnchtt replied:
I think global community = global population of humans.

pbuttigieg commented on src/sdgio-edit.owl:
```diff
@@ -158,6 +159,62 @@ Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020232>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020335>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020337>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020338>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020339>))
```
@Mttbnchtt make sure to link this to: 

http://purl.unep.org/sdg/SDGIO_00000071 (SDGIO):
By 2030, ensure that all men and women, in particular the poor and the vulnerable, have equal rights to economic resources, as well as access to basic services, ownership and control over land and other forms of property, inheritance, natural resources, appropriate new technology and financial services, including microfinance 

http://purl.unep.org/sdg/SDGIO_00000115 (SDGIO):
Undertake reforms to give women equal rights to economic resources, as well as access to ownership and control over land and other forms of property, financial services, inheritance and natural resources, in accordance with national laws 

http://purl.unep.org/sdg/SDGIO_00000172 (SDGIO):
By 2030, achieve the sustainable management and efficient use of natural resources

Mttbnchtt replied:
I tried several ways, but I am puzzled about what to do. On the one hand, we have natural resource, which is an independent continuant. On the other hand, we have three individuals, which do not belong to the class. Should I link them via some relations? However, I did not find any relation that seems ok, especially given that we have individuals. What would you suggest? Thanks very much.

pbuttigieg replied:
![image](https://user-images.githubusercontent.com/2276146/61394242-fcce2180-a8c2-11e9-8547-604e5833965a.png)


Something like that

Mttbnchtt replied:
I think I did it.

pbuttigieg commented on src/sdgio-edit.owl:
```diff
@@ -158,6 +159,62 @@ Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020232>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020337>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020338>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020339>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020341>))
```
As stated in the comment, this cross-links to the discussion on https://github.com/EcologicalSemantics/ecocore/issues/108

Mttbnchtt replied:
I am not sure whether there is something to change in APs for this item. If yes, what?

pbuttigieg replied:
No, it's fine just a note.

pbuttigieg commented on src/sdgio-edit.owl:
```diff
@@ -158,6 +159,62 @@ Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020232>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020338>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020339>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020341>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020342>))
```
Rephrasing def

Mttbnchtt replied:
Thanks very much.

pbuttigieg commented on src/sdgio-edit.owl:
```diff
@@ -158,6 +159,62 @@ Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020232>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020339>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020341>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020342>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020343>))
```
As above, will be linked to decisions on https://github.com/EcologicalSemantics/ecocore/issues/108

Mttbnchtt replied:
Link inserted.

pbuttigieg commented on src/sdgio-edit.owl:
```diff
@@ -158,6 +159,62 @@ Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020232>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020341>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020342>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020343>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020349>))
```
The definition and label are basically the same here, with no supporting semantics to help. I've tried to expand the def slightly, but we need more input on what this actually entails.

pbuttigieg replied:
You are also no using the genus/differentia phrasing in these defs, leading to a lot of repetition. Please fix that globally.

Mttbnchtt replied:
New definition: "A planned process during which an agent or group of agents improves the policies of some organization O so that O increases its ability to contribute to some collective planned process that a group of organizations to which O belongs implements." The SRF document and other CGIAR documents that I know are not more specific. Do you want me to open an issue?

pbuttigieg replied:
> You are also not using the genus/differentia phrasing in these defs, leading to a lot of repetition. Please fix that globally.

This is not addressed.

> "A planned process during which an agent or group of agents improves the policies of some organization O so that O increases its ability to contribute to some collective planned process that a group of organizations to which O belongs implements."

We're not going to use variables like "O" in SDGIO defs as the audience is not familiar with these.

planned process to improve organizational policies to increase the capacity of a partnership of organisations to work towards a common objective
"A planned process to improve organizational policies during which an agent increases its ability to contribute to a planned process collectively pursued by a group of organisations "

Mttbnchtt replied:
Thanks for rephrasing the definition. 

About the genus/differentia: I thought the way to apply that style in the definition of processes was to say _An A during which B happens_. Is that not the case? I thought that, among other places, that was said in issue #183.

Mttbnchtt replied:
Rephrased definitions with the genus being the immediate superclass.

pbuttigieg committed:
some revisions for #170

pbuttigieg commented:
The following still need review 
```
Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020359>))
Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020360>))
Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020361>))
Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020362>))
Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020364>))
Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020365>))
Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020366>))
Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020367>))
Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020368>))
Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020369>))
Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020370>))
Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020371>))
Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020372>))
Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020374>))
Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020375>))
Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020376>))
Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020377>))
Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020378>))
Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020379>))
Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020380>))
Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020383>))
Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020384>))
Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020390>))
Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020393>))
Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020394>))
Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020395>))
Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020397>))
Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020398>))
Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020399>))
Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020400>))
Declaration(NamedIndividual(<http://purl.unep.org/sdg/SDGIO_00000035>))
Declaration(NamedIndividual(<http://purl.unep.org/sdg/SDGIO_00000036>))
Declaration(NamedIndividual(<http://purl.unep.org/sdg/SDGIO_00000037>))
@@ -412,6 +469,19 @@ Declaration(NamedIndividual(<http://purl.unep.org/sdg/SDGIO_00020302>))
Declaration(NamedIndividual(<http://purl.unep.org/sdg/SDGIO_00020303>))
Declaration(NamedIndividual(<http://purl.unep.org/sdg/SDGIO_00020304>))
Declaration(NamedIndividual(<http://purl.unep.org/sdg/SDGIO_00020305>))
Declaration(NamedIndividual(<http://purl.unep.org/sdg/SDGIO_00020346>))
Declaration(NamedIndividual(<http://purl.unep.org/sdg/SDGIO_00020351>))
Declaration(NamedIndividual(<http://purl.unep.org/sdg/SDGIO_00020353>))
Declaration(NamedIndividual(<http://purl.unep.org/sdg/SDGIO_00020357>))
Declaration(NamedIndividual(<http://purl.unep.org/sdg/SDGIO_00020373>))
Declaration(NamedIndividual(<http://purl.unep.org/sdg/SDGIO_00020381>))
Declaration(NamedIndividual(<http://purl.unep.org/sdg/SDGIO_00020382>))
Declaration(NamedIndividual(<http://purl.unep.org/sdg/SDGIO_00020387>))
Declaration(NamedIndividual(<http://purl.unep.org/sdg/SDGIO_00020388>))
Declaration(NamedIndividual(<http://purl.unep.org/sdg/SDGIO_00020389>))
Declaration(NamedIndividual(<http://purl.unep.org/sdg/SDGIO_00020391>))
Declaration(NamedIndividual(<http://purl.unep.org/sdg/SDGIO_00020392>))
Declaration(NamedIndividual(<http://purl.unep.org/sdg/SDGIO_00020396>))
``` 

@Mttbnchtt see if you can improve the definitions of the classes above based on the previous review. Avoid repeating the label and use the genus/differentia approach more effectively so you don't have to repeat quite so much.

pbuttigieg committed:
Addresses #178

Mttbnchtt committed:
First try to fix definition of process to reduce proverty

Mttbnchtt committed:
More to fix definition of process to reduce poverty

Mttbnchtt committed:
More to fix definition of process to reduce poverty/2

Mttbnchtt commented on src/sdgio-edit.owl:
```diff
@@ -158,6 +159,62 @@ Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020232>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020318>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020319>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020320>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020325>))
```
What about: A planned process during which an agent causes or contributes to the protection of life, its diversity, natural resources, natural ecosystems, and human communities from climate change?

This definition tries to mention, in broad strokes, everything that climate change adversely affects.

pbuttigieg replied:
> A planned process during which an agent causes or contributes to the protection of life, its diversity, natural resources, natural ecosystems, and human communities from climate change

Label: planned process to respond to climate change

No, still too specific. A "response" can be anything, not just protection.

Mttbnchtt replied:
New: A planned process to respond to environmental challenges during which an agent causes or contributes to taking actions concerning the impact of climate change on living beings, life diversity, natural resources, natural ecosystems, or human communities.

marieALaporte replied:
"A planned process to respond to environmental challenges during which an agent takes actions to address the impact of climate change on living beings, life diversity, natural resources, natural ecosystems, or human communities." @pbuttigieg would this definition work?

Mttbnchtt committed:
New definition process to respond to climate change

Mttbnchtt commented on src/sdgio-edit.owl:
```diff
@@ -158,6 +159,62 @@ Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020232>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020319>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020320>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020325>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020326>))
```
Great. Thanks very much.

Mttbnchtt commented on src/sdgio-edit.owl:
```diff
@@ -158,6 +159,62 @@ Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020232>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020320>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020325>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020326>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020327>))
```
Thanks a lot.

Mttbnchtt commented on src/sdgio-edit.owl:
```diff
@@ -158,6 +159,62 @@ Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020232>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020325>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020326>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020327>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020328>))
```
very good

Mttbnchtt commented on src/sdgio-edit.owl:
```diff
@@ -158,6 +159,62 @@ Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020232>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020326>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020327>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020328>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020329>))
```
Yes, but what about: A planned process during which an agent attempts to mitigate OR REVERSE the adverse effects of environmental dynamics upon natural or anthropised ecosystems, or to develop adaptation strategies to cope with these effects?

pbuttigieg replied:
We should be more general:

"A planned process to respond to global challenges during which an agent takes action as a result of environmental dynamics impacting on natural or anthropised ecosystems."

Note that I'm using the genus/differentia model I asked you to stick to globally.

pbuttigieg replied:
Subclasses would deal with mitigation, adaptation, or "reversal" (not a fan of this idea, restoration might be better)

Mttbnchtt replied:
Added: "A planned process during which an agent responds to global challenges during which an agent takes action as a result of environmental dynamics impacting on natural or anthropised ecosystems."

Mttbnchtt replied:
Added,

marieALaporte replied:
the double 'during which' is not needed as the first one deals with what's already defined in the superclass
changed to "A planned process to respond to global challenges during which an agent takes action as a result of environmental dynamics impacting on natural or anthropised ecosystems."

Mttbnchtt committed:
Corrected typo

Mttbnchtt commented on src/sdgio-edit.owl:
```diff
@@ -158,6 +159,62 @@ Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020232>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020327>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020328>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020329>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020331>))
```
Done with issue #193.

Mttbnchtt committed:
Improved definitions and removed fairness class

Mttbnchtt committed:
Fixed ORCID typo

Mttbnchtt committed:
revision of process classes

pbuttigieg commented:
@Mttbnchtt it seems you have a couple of orphaned instances:
![image](https://user-images.githubusercontent.com/2276146/61390217-a230c780-a8ba-11e9-809c-2470d44d7f9e.png)

pbuttigieg committed:
adding has disposition relation

Mttbnchtt committed:
Review of some process classes

Mttbnchtt committed:
Revision of process classes /2

Commit:
deleted repeated individual

Commit:
deleted repeated individual

Commit:
adding comments on poverty

Mttbnchtt committed:
added individual CGIAR process to reduce poverty

Mttbnchtt committed:
comment on CGIAR planned process to reduce poverty

Mttbnchtt commented:
I did my best to make the definitions of process classes with already descriptive labels more informative. The problem is that those classes derive directly from sub-IDOs and we do not have definitions of those. CGIAR takes the label of sub-IDOs to be clear enough, at least in most cases, to understand what it describes. I changed something anyway, hopefully improving the situation. When (if) our requests for clarification receive an answer, I will be able to do more.

pbuttigieg committed:
adding example instance-level type axioms for SDGIO_00000071

Mttbnchtt committed:
Added aboutness axiom to individuals

pbuttigieg committed:
added import target for envo_import.owl

pbuttigieg committed:
Merge branch 'issue-169' of https://github.com/SDG-InterfaceOntology/sdgio into issue-169

Mttbnchtt committed:
Fixed process class

pbuttigieg committed:
adding instance axioms to begin linkage, import extension for energy access

pbuttigieg committed:
importing power plants

pbuttigieg committed:
expanding PATO import

pbuttigieg committed:
actualising new PATO import

pbuttigieg committed:
forming ENVO cross-links

pbuttigieg committed:
actualising ENVO import

pbuttigieg committed:
more ENVO cross-links and imports

pbuttigieg committed:
actualising ENVO import

Mttbnchtt committed:
Merge branch 'issue-169' of https://github.com/SDG-InterfaceOntology/sdgio into issue-169

Mttbnchtt committed:
Fix ORCID

Mttbnchtt committed:
Aristotelianize process classes

Mttbnchtt committed:
Aristotelianizing process classes /2

Mttbnchtt committed:
Aristotelianizing process classes /3

Mttbnchtt committed:
Aristotelianizing process classes /4

Mttbnchtt commented:
I think I implemented the requested changes. @marieALaporte @pbuttigieg: when you have time, could you please review the PR again? 

How to add pictures here? I took a screenshot that I would like to share.

Mttbnchtt committed:
Small edits

pbuttigieg commented:
Hi @Mttbnchtt 
@marieALaporte and I are working on this together this week.

> Moreover, we have axioms with canceled-yet-visible terms, like in the following case. Is it ok? What do axioms like these mean?

We should not use obsolete terms. @marieALaporte found the thread related to this one:
https://github.com/oborel/obo-relations/issues?utf8=%E2%9C%93&q=has+active+participant

pbuttigieg commented:
@Mttbnchtt please make sure you're using the latest version of Protege (v 5.5.0)

marieALaporte committed:
added id range

pbuttigieg committed:
Further revisions on process hierarchy - capacity building in focus

marieALaporte committed:
finalisation of capacity building hierarchy and initial review of management process

marieALaporte commented on src/sdgio-edit.owl:
```diff
@@ -158,6 +164,61 @@ Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020232>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020349>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020359>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020360>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020361>))
```
- change spelling to British English 
- moved SDGIO_00020362 to capacity development following its definition and linking it through parthood to 'planned process to apply organisational policies' under management processes

marieALaporte commented on src/sdgio-edit.owl:
```diff
@@ -158,6 +161,61 @@ Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020232>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020342>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020343>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020349>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020359>))
```
the revision of this class and related classes will continue in subsequent edits

marieALaporte committed:
testing management hierarchy

marieALaporte committed:
fixed CGIAR instance label

marieALaporte committed:
finalisation of the development of research capacity

marieALaporte committed:
reorganisation of the capacity development branch

marieALaporte committed:
more enhancements to the capacity development branch

marieALaporte committed:
fixed typo

marieALaporte committed:
example of linking SRF to SDG and further development to the capacity development hierarchy

marieALaporte committed:
added annotation property for SDG ID

marieALaporte committed:
removed language tag from SDG IDs

Commit:
sdgio-edit file to start adding information about IDOs - Aug 26 2019

Commit:
commiting sdgio-edit.obda and sdgio-edit.properties and all the other files

marieALaporte commented on src/sdgio-edit.owl:
```diff
@@ -158,6 +164,63 @@ Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020232>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020318>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020319>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020325>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020327>))
```
- moving this process under 'development of resource acquisition capacity' as it is about access to natural resources
- updated definition

marieALaporte commented on src/sdgio-edit.owl:
```diff
@@ -158,6 +164,63 @@ Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020232>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020308>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020309>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020310>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020311>))
```
removing this process as it is very specific. 
Instances of this class will be mapped to the 'development of natural resource acquisition capacity' class and to agricultural system

marieALaporte committed:
revised definitions

Commit:
adding the class development of capacity to increase resilience and related classes

Commit:
adding obda and properties

Commit:
classes concerning market access and IDO 2

Mttbnchtt committed:
fixing IRIs

Mttbnchtt committed:
processes concerning IDO 2 and IDO 3

Mttbnchtt committed:
added rocesses for IDO 4 and 5 and IDO C1

Mttbnchtt committed:
axioms for improve diet

Mttbnchtt committed:
definition of process for IDO 5

Mttbnchtt committed:
IDO 7 processes

Mttbnchtt committed:
more definitions IDO 7

Mttbnchtt committed:
process human health subclass of process animal health

Mttbnchtt committed:
IDO 7 processes

Mttbnchtt committed:
processes IDO 9-10

Mttbnchtt committed:
processes for all remaining IDOs except IDOs C and D

Mttbnchtt committed:
remove incorrectly placed definition

Commit:
new class

Mttbnchtt committed:
definition of resilience process

Mttbnchtt committed:
state and smallholder

Mttbnchtt committed:
IDO 1, 2, and 3

Mttbnchtt committed:
IDO 3 and 4

Mttbnchtt committed:
access to market

Mttbnchtt committed:
axioms about market and agricultural market

Mttbnchtt committed:
import farm and market from ENVO

Mttbnchtt committed:
systems

Mttbnchtt committed:
IDO terms inserted

Mttbnchtt committed:
IDOs processes

Mttbnchtt committed:
removing has participant from organization

Mttbnchtt committed:
definition of economic sector

Mttbnchtt committed:
definition of economic sector 2

Mttbnchtt committed:
fixing axioms

Mttbnchtt committed:
ratio from PATO

Mttbnchtt committed:
new terms

Mttbnchtt committed:
market

Mttbnchtt committed:
enabling environment, economic system, ENVO import

Mttbnchtt committed:
health

Mttbnchtt committed:
IDO 7-9 and definition of enhancement

Mttbnchtt committed:
IDO A-D

marieALaporte committed:
remove a sdgio-edit.properties

marieALaporte committed:
updated pato imports

marieALaporte committed:
Delete sdgio-edit.obda

marieALaporte committed:
Merge pull request #225 from SDG-InterfaceOntology/issue-201

Issue 201

marieALaporte committed:
made all_imports

Mttbnchtt committed:
correct grammar in definition of inclusivity process

Mttbnchtt committed:
added comment

Mttbnchtt committed:
ido 9

Mttbnchtt committed:
'has part' some ('Homo sapiens' or organization)

Mttbnchtt committed:
'has part' some ('Homo sapiens' or organization) for planned process to respond to global challenges

Mttbnchtt committed:
add equity and inclusivity disposition links to processes

Mttbnchtt committed:
smallholding

Mttbnchtt committed:
fix processes concerning food

Mttbnchtt committed:
connect CGIAR planned process to increment the capacity of CGIAR partner organization to process enable CGIAR national partners

pbuttigieg commented:
Merging to kick off a new round of development
